bindings/pydeck/examples/text_layer.py

```diff
@@ -0,0 +1,32 @@
+import pydeck as pdk
+import pandas as pd
+
+TEXT_LAYER_DATA = ('https://raw.githubusercontent.com/uber-common/'
+                   'deck.gl-data/master/website/bart-stations.json')
+df = pd.read_json(TEXT_LAYER_DATA)
+
+# Define a layer to display on a map
+layer = pdk.Layer(
+    'TextLayer',
+    df,
+    pickable=True,
+    get_position='coordinates',
+    get_text='name',
+    get_size=32,
+    get_angle=0,
+    get_text_anchor='middle',
```

Comment:
This isn't well-documented but if you're passing a string constant use double quotes. There's no field named `middle` in the data, so you likely want to pass the constant `"middle"`.